drivers/serial/uart_nrfx_uarte.c

```diff
@@ -73,8 +73,12 @@ struct uarte_async_cb {
 	void *user_data;
 
 	const uint8_t *tx_buf;
+	const uint8_t *xfer_buf;
+	size_t xfer_len;
 	volatile size_t tx_size;
```

Comment:
Nit: `tx_buf` and `tx_size` are coupled, so I think it's better to keep their definitions next to each other.